### DISCUSSION TITLE
The Gauntlet of Death race map.

### DISCUSSION THREAD

**Discussion by SniperGoth** [redeclipse/base#1211] [Maps and Content Showcase]
A very simple race map, heavily inspired by "The Hardest Game", which has 5 rooms with a lot of obstacles to conquer.

![room1](https://user-images.githubusercontent.com/24359960/94328506-93977300-ff89-11ea-8d4f-98d633b57d5f.gif)
![room 2](https://user-images.githubusercontent.com/24359960/94328546-d9543b80-ff89-11ea-9b2e-740e56beeb22.gif)
![room 3](https://user-images.githubusercontent.com/24359960/94328595-318b3d80-ff8a-11ea-8afd-ac32add5b4c1.gif)
![room4](https://user-images.githubusercontent.com/24359960/94328634-78793300-ff8a-11ea-971b-a196a15cb04c.gif)
![room5](https://user-images.githubusercontent.com/24359960/94328669-c68e3680-ff8a-11ea-8954-f51212c163e7.gif)

[thegauntlet.zip](https://github.com/redeclipse/world/files/5286106/thegauntlet.zip)

RE 2.1 and onwards only.

**Comment by qreeves** [comment]
Are you gonna PR these to maps?

**Comment by SniperGoth** [comment]
If you want, i can PR them, sure. That's mostly your call, if you feel like it fits the game.
